test/integration/integration.py

```diff
@@ -25,7 +25,9 @@
 
 
 class IntegrationTest(unittest.TestCase):
-    FIXTURE_FOLDER = 'fixtures'
+    CWD = os.path.abspath(os.path.dirname(__file__))
+    FIXTURE_FOLDER = CWD + '/fixtures'
```

Comment:
@vlad-isayko what do you mean that this solution is not cross-platform? I tested this code on Linux. Do you want to say that it doesn't work on Windows or OS X?

Reply:
Yes, problems with this will occur on Windows. I believe that because of the problem with the paths, we should not be tied to a specific platform, especially since this is easily solved by changing a couple of lines.

Reply:
@vlad-isayko do you believe or do you know? :)

I bet 5% of my total monthly income that this modification don't make tests fail on Windows. We can settle this in DAI if you've got Metamask installed. You can support me with your offer, but it is not required. It is just for your incentive to run it. )

Reply:
Yes indeed in this particular case with open the error does not fall thanks to the hack with os.altsep. But still, it’s a bad tone to leave it in this form, since in the future, use may change to something that this hack will not support. And yes, this is from the category of not critical things, but rather from the consistency and purity of the code.

Reply:
@vlad-isayko from my previous life, Linux path slashes did not create problems on Windows, unless information about paths is shared with Windows programs. I agree that the code is less readable when people are stumbling on it and have to think about the usage consequences. Software engineering is not programming.

I will fix that.

Reply:
Done.